### ISSUE TITLE
Use the latest jekyll-assets version

### ISSUE THREAD

**Issue by kinueng** [OpenLiberty/openliberty.io#73]
In issue #70, I had to downgrade the version of `jekyll-assets` to fix a hard build break.  We need to figure out whether this is an issue we need to raise in `jekyll-assets` or whether this is our build environment.

**Comment by kinueng** [comment]
I did some research and we need to change the Liquid code to follow the new Liquid tagging used by `jekyll-assets` version 3.

See https://envygeeks.io/2017/11/21/jekyll-assets-3-released#one-tag-to-rule-them-all

**Comment by steven1046** [comment]
Some notes:
Jekyll 4.0 doesn't work with Jekyll-assets 3.0.12 so we had to keep using jekyll 3.8.6.
I couldn't get all of the ruby gem versions to build together (even after compatibility). To get past a few errors, I had to change sprockets to version 3.7 and uglifier to 4.1 but jekyll serve still fails with no helpful error with with --trace and --verbose mode on.

```/.rvm/gems/ruby-2.5.0/gems/json-minify-0.0.3/lib/json/minify.rb:45:in `minify': Unable to pre-scan string: \\", (SyntaxError)```
